src/core/cff_parser.js

```diff
@@ -1556,7 +1556,9 @@ class CFFCompiler {
   }
 
   compileHeader(header) {
-    return [header.major, header.minor, header.hdrSize, header.offSize];
+    // `header.hdrSize` can be any value but we only write 4 values
+    // so header size is 4.
```

Comment:
Please mention that we're doing this to appease OTS, hence the following would be a nice addition to this comment:
```javascript
    // so the header size must be 4 (prevents OTS from rejecting the font).
```